TheMovieDB/.swiftlint.yml

```diff
@@ -16,7 +16,7 @@ excluded:
   - Project/R.generated.swift
 
 force_cast: error
-force_try: error
+force_try: errr
```

Comment:
there is a missing "o"

Reply:
[Splash Screen Fixed](https://github.com/agustingodoyc/TheMovieDB/pull/2/commits/47fb8e3c3fcc65019a7ede66e0e23d51c5dd707b)